test/core-flags/context-flag.test.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
 
 const { resolve } = require('path');
-const { run, isWindows } = require('../utils/test-utils');
+const { run, isWindows, isWebpack5 } = require('../utils/test-utils');
 
 describe('--context flag', () => {
     it('should allow to set context', () => {
         const { stderr, stdout, exitCode } = run(__dirname, ['--context', './']);
 
         expect(stderr).toBeFalsy();
         expect(exitCode).toBe(0);
-        if (isWindows) {
-            expect(stdout).toContain('core-flags');
+        if (isWindows && !isWebpack5) {
+            expect(stdout).toContain(`context: 'D:\\\\a\\\\webpack-cli\\\\webpack-cli\\\\test\\\\core-flags'`);
```

Comment:
```
expect(stdout).toContain(`context: '`${path.resolve('./core-flags)}'`)
```

Reply:
that works for some windows check but also fails on some windows checks. 😞

Reply:
Where is it not work? Having absolute path in tests is really bad idea

Reply:
Issue with no of `\`

`{path.resolve('./core-flags)} = D:\\a\\webpack-cli\\webpack-cli\\test\\core-flags'`

https://github.com/webpack/webpack-cli/runs/1346870398?check_suite_focus=true

![20201103_185442](https://user-images.githubusercontent.com/46647141/97990766-6ea6e300-1e06-11eb-942f-605cb92db64f.jpg)

Reply:
Just replace `\` on `\\` after `path.resolve` and it should work

Reply:
```suggestion
            const windowsPath = resolve(__dirname, './').replace(/\\/g, '\\\\\\\\');
            expect(stdout).toContain(windowsPath);
```